exoplayer-flac/src/main/jni/flac_parser.cc

```diff
@@ -22,6 +22,7 @@
 
 #include <cassert>
 #include <cstdlib>
+#include <cstring>
```

Comment:
For some reason the compiler complained about a missing string import. It was a quick fix from Android studio.